backend/src/main/java/ca/bc/gov/backendstartapi/dto/FavouriteActivityUpdateDto.java

```diff
@@ -0,0 +1,13 @@
+package ca.bc.gov.backendstartapi.dto;
+
+import ca.bc.gov.backendstartapi.entity.FavouriteActivityEntity;
+import jakarta.validation.constraints.NotBlank;
+
+/**
+ * This record represents a dto when updating a {@link FavouriteActivityEntity}.
+ *
+ * @param highlighted A boolean representing if the activity is highlighted
+ * @param enabled a boolean representing if the activity is enabled
+ */
+public record FavouriteActivityUpdateDto(
+    @NotBlank boolean highlighted, @NotBlank boolean enabled) {}
```

Comment:
Yup. We should go with `NotNull` and `Boolean` instead of primite `boolean`. Because using primitive values will aways considering the value as `false` when not existing or not present in the request body.